generator/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "pest_generator"
+description = "pest's derive macro"
+version = "2.0.0"
+authors = ["Dragoș Tiselice <dragostiselice@gmail.com>"]
+homepage = "https://pest-parser.github.io/"
+repository = "https://github.com/pest-parser/pest"
+documentation = "https://docs.rs/pest"
+keywords = ["pest", "parser", "peg", "grammar"]
+categories = ["parsing"]
+license = "MIT/Apache-2.0"
+readme = "_README.md"
+
+[lib]
```

Comment:
Does it need name here?

Reply:
I just included it since I was modifying `pest_derive` which has a lib.name set. I'll remove it here since it's redundant though.

Reply:
Looks like I did a poor job of keeping Cargo files clean. 😓